content/docs/command-reference/repro.md

```diff
@@ -95,7 +101,7 @@ only execute the final stage.
 
 - `-s`, `--single-item` - reproduce only a single stage by turning off the
   recursive search for changed dependencies. Multiple stages are executed
-  (non-recursively) if multiple stage files are given as `targets`.
+  (non-recursively) if multiple stages are given as `targets`.
 
 - `-c <path>`, `--cwd <path>` - directory within the project to reproduce from.
   Instead of using `--cwd`, one can alternately specify a target in a
```

Comment:
Like I just suggested, I'd like to test whether this option even makes provides any value now that dvc repro accepts directories as targets. But even if it does, the paragraph is outdated, for example "specify a target in a subdirectory as `path/to/target.dvc`" — `target.dvc` is not a subdir.

Reply:
> Obviously they are different... I think dvc repro ./ would give an error...
From https://github.com/iterative/dvc.org/pull/1572#issuecomment-659879072

I don't see how it's obvious. Did you test each option @sarthakforwet ? You are making lazy assumptions. Please share your process and results. Thanks

Reply:
> I don't see how it's obvious

Oh sorry! Maybe I interpreted the review incorrectly but I think you are asking for this - 

![image](https://user-images.githubusercontent.com/46163196/87754346-4b8a0080-c822-11ea-829d-43b35dc4a20f.png)

Thanks

Reply:
I see. You are correct then. The thing is you can actually use `dvc repro path/to/dir` if `path/to/dir` is tracked by DVC. That's what I'd like you to play with please, compare using or not using `-c` and `-R` in that scenario, and find a good way to express all your findings in the description; as well, probably, as the new **Specific files or directories** example you're suggesting.

Reply:
If you haven't done this, let's leave it for a separate PR @sarthakforwet

Reply:
Same for the note on parallelism (https://github.com/iterative/dvc.org/pull/1572#issuecomment-660263924).

Reply:
>  The thing is you should be able to use dvc repro path/to/dir (without flags) if path/to/dir is tracked by DVC,

We can specify the corresponding `.dvc` file of the DVC tracked directory in `dvc repro`  (which would update any change in the directory) but not the directory name itself. Moreover we cannot build a pipeline in DVC tracked directories ( as per my experimentation dunno if documented already). Also I think directories have no logical use as a dependency in a pipeline. (although we can add them if we wish). And hence I think it is not incorrect if `dvc repro` does not take directories as targets (without flags).

Since we cannot specify a directory in `dvc repro`, to reproduce pipelines inside a directory/sub-directory (which is untracked), we make use of -c and -R options.

However no example in the doc for using `dvc repro` with a specific `target` stage or .dvc file is provided and hence I think this should be done. WDYT @jorgeorpinel?

Reply:
> We can specify the corresponding .dvc file ... but not the directory name itself

(or a stage name). OK got it. I think you're right. Except when you use `-R`. 
- [ ] This should be clarified in the -R option desc: It expands what the `targets` argument accepts.

> we cannot build a pipeline in DVC tracked directories

I never thought about that... I guess it's to be expected. May need a core repo issue, will think about it 🧠 thanks

> I think directories have no logical use as a dependency in a pipeline

Why not? Of course you can use a directory as a dependency. They can also be outputs. I don't see how this affects our conversation though.

> to reproduce pipelines inside a directory/sub-directory (which is untracked), we make use of -c and -R options

And -P.

> - [ ] no example in the doc for using dvc repro with a specific target stage or .dvc file is provided and hence I think this should be done

Agree. I'll create an issue about these 2 check boxes, ans possibly something else... I'll tag you there. Thanks!

Reply:
See #1632

Reply:
> (or a stage name)

I am referencing DVC tracked directories specifically in [#1572(review)](https://github.com/iterative/dvc.org/pull/1572#discussion_r460402286) and I think "stage name" should not be mentioned with that..

> And -P.

Yeah sure, but I was talking specifically in comparison of `dvc repro` with/without `-R` and `-c` options.

> Agree. I'll create an issue about these 2 check boxes, ans possibly something else...

Thanks!

Reply:
> I am referencing DVC tracked directories specifically

Stages in dvc.yaml can output directories too. It't not exclusive to .dvc files.